Take a stab at a timeline for WebTransport

dontcallmedom opened the pull request:
as a first guess, pending input from others

dontcallmedom committed:
Take a stab at a timeline for WebTransport

tidoust reviewed (commented):
Probably need to update "Expected completion" date in 2.1 as well.

Don't we usually try to make things fit within the charter timeframe?
Meaning something like: "Expected completion: Q3 2022", and CR a bit earlier on, say "April 2020".

dontcallmedom committed:
Update based on comments

dontcallmedom commented:
I've updated from June 2022 to Q2 2022 for CR, and added completion date of Q1 2023 - I don't think we're expected to have the work fit in a single charter (given that we're also not expected to charter groups for more than 2 years at once)
